src/main/webapp/data-visualization.js

```diff
@@ -296,3 +302,47 @@ function setStyle(isCountyQuery) {
     amChartsDiv.style.display = 'block';
   }
 }
+
+function initDataTable(isCountyQuery) {
+  const table = document.createElement('table');
+  table.innerHTML = '';
+  const tableRow = document.createElement('tr');
+  const region = document.createElement('th');
+  const value = document.createElement('th');
+  region.innerText = isCountyQuery ? 'County' : 'State';
+  value.innerText = 'Population';
+  tableRow.appendChild(region);
+  tableRow.appendChild(value);
+  table.appendChild(tableRow);
+  return table;
+}
+
+/**
+ * Create and return an HTML table using the data in dataArray.
+ * Uses first row in dataArray as table headers.
+ */
+function appendRowToDataTable(table, locationName, population) {
```

Comment:
The data doesn't seem to be sorted by name or value. This might make it little difficult to read.

How about sorting the data either during the append or during display?